### PR TITLE
add a job that allows testing moving the pull-kubernetes-e2e-gce-gpu …

### PR DESCRIPTION
**Pull request by BenTheElder**
…job to prow

see #4610 for motivation.

### CODE REVIEW

**Comment by krzyzacy** [comment]
(fix pylint :-) )

**Comment by BenTheElder** [comment]
AAAAAH trailing whitespace on an empty line!
Hopefully pylint is happy this time :roll_eyes:

File: prow/config.yaml [hunk 1]
```diff
@@ -357,6 +357,49 @@ presubmits:
     context: pull-kubernetes-e2e-gce-gpu
     rerun_command: "/test pull-kubernetes-e2e-gce-gpu"
     trigger: "(?m)^/test( all| pull-kubernetes-e2e-gce-gpu),?(\\s+|$)"
+  - name: pull-kubernetes-e2e-gce-gpu-prow
```
Comment (ixdy) [inline prow/config.yaml]: I think we want to set max_concurrency on this job, too, since it's using a single project.

Comment (BenTheElder) [reply prow/config.yaml]: this job is manually triggered so it should be fine. I only intend to trigger it on the testing PR. If all goes well I will follow up with `branches: -master` defaulting to the prow variant.

File: jobs/config.json [hunk 2]
```diff
@@ -10428,6 +10428,27 @@
...
+    "args": [
+      "--build=bazel",
+      "--cluster=",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/pull-kubernetes-e2e.env",
+      "--env-file=jobs/env/pull-kubernetes-e2e-gce-gpu.env",
+      "--extract=local",
+      "--gcp-project=k8s-jkns-pr-gce-gpus",
+      "--gcp-zone=us-west1-b",
+      "--mode=docker",
```
Comment (BenTheElder) [inline jobs/config.json]: this however needs to be removed! agh
@krzyzacy did we miss anything else?

Comment (BenTheElder) [reply jobs/config.json]: fixed in #4615

**Pull request merged**